torchmetrics/functional/image/total_variation.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+
+def total_variation(img: torch.Tensor) -> torch.Tensor:
+    """Computes total variation loss.
+
+    Adapted from https://github.com/jxgu1016/Total_Variation_Loss.pytorch
+    Args:
+        img (torch.Tensor): A NCHW image batch.
+
+    Returns:
+        A loss scalar value.
+    """
+
+    def tensor_size(t):
+        return t.size()[1] * t.size()[2] * t.size()[3]
+
+    _height = img.size()[2]
+    _width = img.size()[3]
+    _count_height = tensor_size(img[:, :, 1:, :])
+    _count_width = tensor_size(img[:, :, :, 1:])
```

Comment:
this should be equivalent with only one call to the shape function
```suggestion
    _batchsize, _channels, _height, _width = img.shape[1:]
    _count_height = _channels * (_height - 1) * _width
    _count_width = _channels * _height * (_width - 1)
```

Reply:
Yes, sure. Why `img.shape[1:]` though? Shouldn't it be just `img.shape`? Shape is a 4-element tuple correct?